.coderabbit.yaml

```
@@ -0,0 +1,65 @@
+language: "ko"
+early_access: false
+
+reviews:
+  profile: "assertive" # 더 적극적인 리뷰로 품질 향상
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)